cmd/console-proxy/src/posix/picocom_map.rs

```diff
@@ -0,0 +1,185 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! picocom-style character remapping; does not support the "... to hex" rules.
+
+use std::{collections::VecDeque, str::FromStr};
+
+use anyhow::{bail, ensure, Error, Result};
+
```

Comment:
Nit: move `mod raw` up here, so I'm not confused where those values come from in the table below.